python/cudf/cudf/tests/test_copying.py

```diff
@@ -302,6 +305,11 @@ def test_series_zero_copy_cow_on():
 
 
 def test_series_zero_copy_cow_off():
+    if get_global_manager() is not None:
+        pytest.skip(
+            "cannot test zero-copy and no-copy-on-write when "
+            "spilling is enabled globally, set `CUDF_SPILL=off`"
+        )
```

Comment:
Is this because with the unification spilling automatically implies something about copy-on-write semantics? I thought most of the copy-on-write functionality in `ExposureTrackedBuffer` is gated behind the associated option, so all that should be happening by default is the tracking, but correct me if I'm wrong.

Reply:
It is because of the  zero-copy assumption in one of the tests.
Moved the skip into the problematic line and added:
```python
            # Since spilling might make a copy of the data, we cannot
            # expect the two series to be a zero-copy of the cupy array
            # when spilling is enabled globally.
```